history-of-quakers/modernized/25-chapter-23.adoc

```diff
@@ -659,16 +700,21 @@ according to an act made in the first year of the reign of king William and quee
 entitled, '`An Act declaring the Rights and Liberties of the Subjects,
 and settling the succession of the Crown.`'
 
+--
+
 An association was also signed by the lords, and both presented to the king,
 and were followed by all the corporations in England.
-See Life of king William, vol.
-III. The dissenters also presented declarations to the king,
+See [.book-title]#Life of king William, Vol.
+III.# The dissenters also presented declarations to the king,
 that had some resemblance with the other.
 But the Quakers professing non-resistance, and an inoffensive behavior,
 could in no way enter into such a league;
 yet to show that they were loyal and faithful to the king,
 they drew up the following declaration, and published it in print:
 
+[.embedded-content-document.address]
+--
+
 The ancient testimony and principle of the people called Quakers renewed,
```

Comment:
Should this be labeled anything?? It was so hard to tell even when I rendered the PDF to see how it looked!!